src/tests/dictionaryConcepts/container/CreateConcept.test.jsx

```diff
@@ -276,20 +289,50 @@ describe('Test suite for dictionary concepts components', () => {
     expect(instance.state.answers).toEqual([{ frontEndUniqueKey: 'unique', id: 'test ID', map_type: 'Q-AND-A' }]);
   });
 
+  describe('handleSetAsyncSelectChange', () => {
+    it('should update the state with sets', () => {
+      const instance = wrapper.find('CreateConcept').instance();
+      instance.handleSetAsyncSelectChange([{
+        frontEndUniqueKey: 'unique', id: 'test ID', map_type: MAP_TYPE.conceptSet, prePopulated: false,
+      }], 'abc');
+      expect(instance.state.sets).toEqual([{
+        frontEndUniqueKey: 'unique', id: 'test ID', map_type: MAP_TYPE.conceptSet, prePopulated: false,
+      }]);
+    });
+  });
+
   it('should add new answer row', () => {
     const instance = wrapper.find('CreateConcept').instance();
     const spy = jest.spyOn(instance, 'addAnswerRow');
     instance.addAnswerRow();
     expect(spy).toHaveBeenCalled();
   });
 
+  describe('addSetRow', () => {
+    it('should correctly call createNewSetRow', () => {
+      const instance = wrapper.find('CreateConcept').instance();
+      createNewSetRowMock.mockClear();
+      instance.addSetRow();
+      expect(createNewSetRowMock).toHaveBeenCalled();
```

Comment:
Are you testing for what you have just directly called above?

Reply:
I'm testing that that function calls another function it is supposed to call.